### PR TITLE
- Updated busy indicator CSS class in different templates to be displ…

### PR DESCRIPTION
**Pull request by agenius-mohammed-ali**
…ayed correctly with new navbar settings

### CODE REVIEW

**Review by cb4ds — CHANGES_REQUESTED** [review]

File: R/ui_helpers.R [hunk 1]
```diff
@@ -536,7 +544,7 @@ ui_tooltip <- function(id,
 
    shiny::span(class = "periscope-input-label-with-tt",
                label,
-               bs4Dash::tooltip(shiny::img(id     = id,
+               tooltip(shiny::img(id     = id,
                                            src    = shiny::isolate(.g_opts$tt_image),
                                            height = shiny::isolate(.g_opts$tt_height),
                                            width  = shiny::isolate(.g_opts$tt_width)),
```
Comment (agenius-jasmine) [inline R/ui_helpers.R:550]: realign

File: R/ui_helpers.R [hunk 2]
```diff
@@ -664,3 +672,38 @@ get_url_parameters <- function(session) {
 
     parameters
 }
+
+
+# Override bs4Dash tooltip function to allow html handling in tooltips
+tooltip <- function(tag, title, placement = c("top", "bottom", "left", "right")) {
```
Comment (agenius-jasmine) [inline R/ui_helpers.R:678]: just for my information how is our tooltip different from the bs4dash tooltip?

**Review by agenius-jasmine — COMMENTED** [review]
Some questions and modifications

File: NEWS.md [hunk 3]
```diff
@@ -1,46 +1,47 @@
-## 0.1.5
-### New Features
+# 0.2.0
+## New Features
 - Introduced application theme configuration file generator RStudio add-in
+- Enabled full usage of bootstrap4 variables customization in `periscope_style.yaml` (refer to [Migrate periscope2 applications to v0.1.5](migrate_to_v0_1_5.html) document
```
[the remark this answers is not in the document]
Comment (agenius-mohammed-ali) [reply NEWS.md:4]: done

File: vignettes/migrate_to_v0_1_5.Rmd [hunk 4]
```diff
@@ -0,0 +1,69 @@
+---
+title: "Migrate periscope2 applications to v0.1.5"
```
[the remark this answers is not in the document]
Comment (agenius-mohammed-ali) [reply vignettes/migrate_to_v0_1_5.Rmd:2]: done

File: R/themeConfigurationsAddin.R [hunk 5]
```diff
@@ -172,14 +171,19 @@ themeBuilder_addin_UI <- function() {
...
+                                                                                                              sep = "<br/><br/>"),
+                                                                                            placement = "bottom"),
                                                            value   = NA))
                 )
             ),
             miniUI::miniTabPanel(
                 "Main Colors",
                 miniUI::miniContentPanel(
-                    shiny::p("Template main colors definition"),
+                    shiny::p("Specify the specific hex codes for the following color words that can be referenced In the app."),
```
[the remark this answers is not in the document]
Comment (agenius-mohammed-ali) [reply R/themeConfigurationsAddin.R:186]: done

File: vignettes/migrate_to_v0_1_5.Rmd [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+  %\VignetteEncoding{UTF-8}
+---
+
+## Objective
+
+Explain the steps to migrate a periscope2 generated applications with periscope2 prior to v 0.1.5 to use new/updated features of periscope2 v.1.5
+
+## Header Navbar
+
+- Now header is divided into 3 elements:
```
[the remark this answers is not in the document]
Comment (agenius-mohammed-ali) [reply vignettes/migrate_to_v0_1_5.Rmd:16]: done

File: vignettes/migrate_to_v0_1_5.Rmd [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+    display: inline-flex;
+    align-items: center;
+}
+```
+
+## periscope_style.yaml
+
+- Now periscope_style can be generated directly from [theme configuration addin Add-ins](themeBuilder_addin.html)
+- The generated file now contains bootstrap 4 variables names directly to be able to manage all available variables
+- The following variables will be the same and no change in them:
```
[the remark this answers is not in the document]
Comment (agenius-mohammed-ali) [reply vignettes/migrate_to_v0_1_5.Rmd:40]: done

File: vignettes/migrate_to_v0_1_5.Rmd [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+  - primary, secondary, success, info, warning, danger, light and dark
+- The following are the mapping between old periscope_style.yaml file keys and new used keys:
```
[the remark this answers is not in the document]
Comment (agenius-mohammed-ali) [reply vignettes/migrate_to_v0_1_5.Rmd:42]: done

File: vignettes/migrate_to_v0_1_5.Rmd [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+  - submenu_background_color -> submenu_bg
+  - submenu_color -> submenu_color
+  - submenu_hover_color -> submenu_hover_color
+  - submenu_background_hover_color -> submenu_hover_bg
+  - submenu_active_color -> submenu_active_color
+  - submenu_active_background_color -> submenu_active_bg
+  - header_color -> "navbar-light-color"
+  - button_background_color -> "button-default-background-color"
+  - button_color -> "button-default-color"
+  - button_border_color -> "button-default-border-color"
```
[the remark this answers is not in the document]
Comment (agenius-mohammed-ali) [reply vignettes/migrate_to_v0_1_5.Rmd:65]: The quotations are used whenever there are special characters in the variables names (i.e the minus sign). That mostly happens with bootstrap theme variables names

File: R/ui_helpers.R (hunk 2 above)
Comment (agenius-mohammed-ali) [reply R/ui_helpers.R:678]: bs4Dash does not allow html tags inside a directly added tooltip in the UI (while they allow it if you added the tooltip programmatically!!!). So, we just allowed using html tags for directly created tooltip tags

File: R/ui_helpers.R (hunk 1 above)
Comment (agenius-mohammed-ali) [reply R/ui_helpers.R:550]: done

**Pull request merged**